package.json

```diff
@@ -2,28 +2,27 @@
   "name": "redux-actions",
   "version": "2.6.5",
   "description": "Flux Standard Action utlities for Redux",
-  "main": "lib/index.js",
-  "unpkg": "dist/redux-actions.js",
-  "module": "es/index.js",
-  "sideEffects": false,
+  "type": "module",
   "scripts": {
     "format": "prettier --write \"**/*.{js,md}\"",
     "format:check": "prettier --list-different \"**/*.{js,md}\"",
-    "build:esm": "cross-env BABEL_ENV=esm babel src --out-dir es",
-    "build:commonjs": "cross-env BABEL_ENV=commonjs babel src --out-dir lib",
-    "build:umd": "rollup -c && es-check es5 dist/redux-actions.min.js",
-    "build": "run-s clean && run-p build:**",
+    "build": "vite build",
     "clean": "rimraf coverage dist es lib",
-    "lint": "xo",
     "prepublishOnly": "run-s build",
-    "test": "jest"
+    "test": "vitest run --coverage",
+    "lint": "xo"
   },
   "files": [
-    "es",
-    "lib",
-    "dist",
-    "src"
+    "dist"
   ],
+  "main": "./dist/redux-actions.umd.js",
+  "module": "./dist/redux-actions.mjs",
+  "exports": {
+    ".": {
+      "import": "./dist/redux-actions.mjs",
+      "require": "./dist/redux-actions.umd.cjs"
+    }
+  },
```

Comment:
The question we should ask ourselves actually is if should we continue supporting CommonJS? ESM seems to be what everyone is exporting now and what is becoming the standard in the future to replace CJS. Apparently supporting both can also lead to dual [package hazard](https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c?permalink_comment_id=3850849#gistcomment-3850849).

Note: ESM can still import CJS, but not vice versa. If you're already using a modern stack, ESM should be support anyway.

Releasing a new major would also be more justified in this case, making it a pure ESM package.

Related: https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c

Reply:
I'm all for a pure ESM package, since this PR doesn't introduce new features we should be good.